src/sources/kubernetes_logs/mod.rs

```diff
@@ -167,6 +168,7 @@ pub struct Config {
     #[serde(default)]
     #[configurable(metadata(docs::type_unit = "seconds"))]
     #[configurable(metadata(docs::examples = 600))]
+    #[configurable(metadata(docs::human_name = "Ignore Older Files"))]
```

Comment:
(Not knowing how this is presented) would it make sense to frame this as `Ignore Files Older Than`?

Reply:
Make sense, updated.